sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryPipeline.java

```diff
@@ -63,6 +74,11 @@ public CompletableResultCode send(
         }
     }
 
+    // Statsbeat is enabled when metric exporter is used
+    public void setStatsbeatModule(StatsbeatModule statsbeatModule) {
+        this.statsbeatModule = statsbeatModule;
+    }
+
```

Comment:
can we inject into constructor and make `statsbeatModule` final?

Reply:
i did that earlier, but then need to change quite a bit in the public class.. azuremonitorexporterbuilder.